docs/guided-tour/basics/upgrade/commands/deploy-k8s-resources.sh

```diff
@@ -20,7 +20,7 @@ echo "creating target"
 echo "target cluster kubeconfig: $TARGET_CLUSTER_KUBECONFIG_PATH"
 outputFile="${TMP_DIR}/target.yaml"
 export namespace="${NAMESPACE}"
-export kubeconfig_path="${TARGET_CLUSTER_KUBECONFIG_PATH}"
+export kubeconfig=`sed 's/^/      /' $TARGET_CLUSTER_KUBECONFIG
```

Comment:
I am afraid you did not copy the closing backtick -- in all examples except the manifest example.

Reply:
good point :-) I have changed this.